wagtail/admin/views/generic/chooser.py

```diff
@@ -145,11 +197,21 @@ def columns(self):
                 "title",
                 label=_("Title"),
                 accessor=str,
-                url_name=self.chosen_url_name,
+                get_url=(
```

Comment:
Should we remove the URL if it's a multiple chooser? If we still have the URL, accidentally clicking on the title after selecting many of the checkboxes will close the modal with only selecting the one clicked.

Reply:
As discussed, have now changed the title into a label for the checkbox in 9621d35b02fda8b659e8220f72e613841a3ffaab.